tests/test_extract_child_mesh.cpp

```diff
@@ -0,0 +1,589 @@
+#include <catch2/catch_test_macros.hpp>
+
+#include <wmtk/Types.hpp>
+#include <wmtk/multimesh/same_simplex_dimension_surjection.hpp>
+#include <wmtk/multimesh/utils/extract_child_mesh_from_tag.hpp>
+#include <wmtk/multimesh/utils/tuple_map_attribute_io.hpp>
+#include <wmtk/operations/tri_mesh/EdgeCollapse.hpp>
+#include <wmtk/operations/tri_mesh/EdgeSplit.hpp>
+#include "tools/DEBUG_TriMesh.hpp"
+#include "tools/DEBUG_Tuple.hpp"
+#include "tools/TriMesh_examples.hpp"
+
+using namespace wmtk;
+using namespace wmtk::tests;
+
+using TM = TriMesh;
+using TMOE = decltype(std::declval<DEBUG_TriMesh>().get_tmoe(
+    wmtk::Tuple(),
+    std::declval<Accessor<long>&>()));
```

Comment:
If I see it correctly, you are neither using `TM` nor `TMOE`. Please remove that.